packages/shared/lib/assets.ts

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2018 - 2023 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+export interface NFTTokenUriMetaData {
+  name: string
+  description?: string
+  image: string
+  attributes?: [
+    {
+      trait_type: string
+      value: string | number | boolean
+    }
+  ]
```

Comment:
This says that `attributes` can either be `undefined` or an instance of an `Array` with **a single** element in position `0` whose value is 

```ts
{
  trait_type: string
  value: string | number | boolean
}
```

Don't we want this instead?
```
attributes?: {
  trait_type: string
  value: string | number | boolean
}[]
```